model/src/main/java/com/ge/predix/acs/model/Policy.java

```diff
@@ -79,8 +96,8 @@ public void setEffect(final Effect effect) {
     @SuppressWarnings("nls")
     @Override
     public String toString() {
-        return "Policy [name=" + this.name + ", target=" + this.target + ", conditions=" + this.conditions + ", effect="
-                + this.effect + "]";
+        return "Policy [name=" + this.name + ", target=" + this.target + ", conditions=" + this.conditions
+                + ", obligationIds=" + this.obligationIds + ", effect=" + this.effect + "]";
```

Comment:
Assigning "conditions=this.conditions" will print the reference/hash of the list. Question is do we need the reference/hash of the conditions or the actual values of the conditions. If we need the actual values this code needs to be changed.

Reply:
decided not to proceed with this